clang/include/clang/Basic/Attr.td

```diff
@@ -4027,6 +4030,12 @@ def SelectAny : InheritableAttr {
   let SimpleHandler = 1;
 }
 
+def HybridPatchable : DeclOrTypeAttr, TargetSpecificAttr<TargetArm64EC> {
+  let Spellings = [Declspec<"hybrid_patchable">, GCC<"hybrid_patchable">];
```

Comment:
I don't see any evidence that GCC supports this attribute, were you aiming for supporting `__attribute__((hybrid_patchable))` and not `[[gnu::hybrid_patchable]]`? If so, that would be a `GNU` spelling instead of a `GCC` spelling, but we typically would use a `Clang` spelling so that `[[clang::hybrid_patchable]]` works.

Reply:
Yes, I was thinking mostly about `__attribute__((hybrid_patchable))` as it's the form that mingw will mostly likely want to use, so I meant `GNU`. `Clang` sounds good to me if proffered, I changed that and added a test for it.